hrjobcontract/tests/phpunit/CRM/Hrjobcontract/Import/Parser/ApiTest.php

```diff
@@ -47,6 +53,22 @@ function setUp() {
     ];
   }
 
+  function createPayScale() {
```

Comment:
From what Omar said in the chat, as long as it implements TransactionalInterface it should be reset after each test.